vispy/visuals/text/text.py

```diff
@@ -463,7 +464,8 @@ def draw(self, transforms):
               transforms.framebuffer_to_render)
         px_scale = (tr.map((1, 0)) - tr.map((0, 1)))[:2]
         self._program.vert['transform'] = transforms.get_full_transform()
-        self._program.vert['text_scale'] = STTransform(scale=px_scale * n_pix)
+        self._text_scale.scale = px_scale * n_pix
+        self._program.vert['text_scale'] = self._text_scale
```

Comment:
This change is only to avoid creating a new transform on every frame (and thus triggering recompiles). Unrelated to the PR, but I came across it while profiling.